geo-types/src/line_string.rs

```diff
@@ -174,7 +174,7 @@ where
     type Point = Point<T>;
 
     fn mbr(&self) -> ::spade::BoundingRect<Self::Point> {
-        let bbox = self.bbox();
+        let bbox = self.bounding_rect();
```

Comment:
nit: seems like you're preferring `let bounding_rect =` elsewhere

Reply:
yep, good catch, thanks